Gandalf-Portal/gandalf-hold

```diff
@@ -1,36 +1,29 @@
 #!/bin/bash
 
-echo "This needs su to run. If it doesn't work... that's why. Run 'su' (the default password is alpine) and then try again."
+# Make sure only root can run our script
+if [[ $EUID -ne 0 ]]; then
+   echo "This script must be run as root. Please run 'su' before (the default password is alpine) and then try again." 1>&2 
+   exit 1
+fi
 
-sleep 2
-
-echo "Gandalf"
+echo "---Gandalf---"
 
 sleep 1
 
 echo "Created by EthanRDoesMC and mehulrao"
 
 sleep 1
-
+# Maybe better make it one script for all?
```

Comment:
Perhaps. Like have the build.sh add in the version-specific stuff. Would certainly make things easier. Great idea!